pkg/util/testingjobs/raycluster/wrappers.go

```diff
@@ -80,6 +80,12 @@ func (j *ClusterWrapper) NodeSelectorHeadGroup(k, v string) *ClusterWrapper {
 	return j
 }
 
+// Set replica count
+func (j *ClusterWrapper) SetReplicaCount(c int32) *ClusterWrapper {
```

Comment:
```suggestion
func (j *ClusterWrapper) WorkerCount(c int32) *ClusterWrapper {
```

Avoid Set/Get in golang https://go.dev/doc/effective_go#getters